contracts/ckb-combine-lock-v2/src/entry.rs

```diff
@@ -0,0 +1,101 @@
+use crate::blake2b::hash;
+use crate::error::Error;
+use alloc::ffi::CString;
+use alloc::vec::Vec;
+use ckb_combine_lock_common::blockchain::{WitnessArgs, WitnessArgsReader};
+use ckb_combine_lock_common::combine_lock_mol_v2::CombineLockWitness;
+use ckb_std::{
+    ckb_constants::Source,
+    ckb_types::{bytes::Bytes, core::ScriptHashType, prelude::*},
+    high_level::{load_script, load_witness_args, spawn_cell},
+};
+use core::result::Result;
+use log::info;
+
+fn parse_execution_args() -> Result<Bytes, Error> {
+    if ckb_std::env::argv().len() == 0 {
+        let script = load_script()?;
+        Ok(script.args().unpack())
+    } else {
+        Ok(ckb_std::env::argv()[0].to_bytes().into())
+    }
+}
+
+fn parse_execution_witness_args() -> Result<WitnessArgs, Error> {
+    if ckb_std::env::argv().len() == 0 {
+        Ok(load_witness_args(0, Source::GroupInput)?)
+    } else {
+        let data = ckb_std::env::argv()[1].to_bytes();
+        match WitnessArgsReader::verify(&data, false) {
+            Ok(()) => Ok(WitnessArgs::new_unchecked(data.into())),
```

Comment:
This argument should be a lock field of witness, not WitnessArgs. Document updated:
> Please note, every element in inner_wintess, actually is a piece of data from lock field of WitnessArgs.